requirements.txt

```diff
@@ -1,8 +1,7 @@
-#rerun
 Mako==0.7.3
 MarkupSafe==0.15
 PyHAML==0.1.9
-bottle==0.12.07
```

Comment:
Bottle needed to be updated. Can't remember the issue. I think related to rtsp/rtmp pull request?